extensions/skins/conversation_v1/player.html

```diff
@@ -30,6 +30,35 @@
                    ng-class="{'conversation-skin-progress-dots-iframed': isIframed, 'conversation-skin-progress-dots-not-iframed': !isIframed}">
     </progress-dots>
 
+    <div ng-if="helpHtml" class="conversation-skin-help-card-container">
+      <div class="conversation-skin-help-card">
+        <img class="conversation-skin-oppia-avatar"
+             ng-src="<[oppiaAvatarImageUrl]>" alt="" style="top: 12px;">
+        <button type="button" class="conversation-skin-close-help-card-button"
+                ng-click="clearHelpHtml()">
+          <i class="material-icons md-18">&#xE5CD;</i>
+        </button>
+        <div class="conversation-skin-help-card-content" angular-html-bind="helpHtml">
+        </div>
+      </div>
+    </div>
+
+    <div ng-if="continueToNextCardHtml" class="conversation-skin-help-card-container">
```

Comment:
Eesh, can this whole second card be consolidated into the first card? This wouldn't be difficult if the help card content were always contained in $scope.helpHtml